controllers/aera_hand_controller_protobuf/hand_grab_sphere_controller.cpp

```diff
@@ -107,12 +107,13 @@ void HandGrabSphereController::run() {
       break;
     }
     auto msg = receive_queue_->dequeue();
-    if (!msg && diagnostic_mode_)
+    if (!msg && diagnostic_mode_ && aera_us >= receive_deadline)
```

Comment:
In diagnostic mode, we only force a wait for input if past the receive_deadline.